lib/fiddle/truffleruby.rb

```diff
@@ -0,0 +1,623 @@
+# frozen_string_literal: true
+# truffleruby_primitives: true
+
+# Copyright (c) 2019, 2024 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 2.0, or
+# GNU General Public License version 2, or
+# GNU Lesser General Public License version 2.1.
+
+module Truffle::FiddleBackend
```

Comment:
We shouldn't define `Truffle` or `Truffle::FiddleBackend` here but something like `Fiddle::TruffleRubyBackend`.

Reply:
You can change `lib/fiddle/truffleruby.rb` in this branch directly.